### PR TITLE
feat: added landing page, carousel component, react-router setup

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
1. Converted HTML ,CSS to React components in landing page (home-page)
2. Added Carousel component
3. React-router setup

### CODE REVIEW

**Review by ashishnnnnn — APPROVED** [review]
Code seems structured and pretty much extensible :+1:

File: src/components/carousel/index.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  position: absolute;
+  opacity: 0;
+  transition: opacity ease-in-out 0.4s;
+}
+
+.hero-bg-img-active {
+  opacity: 1;
+}
+
+/* .bg-img-container {
```
Comment (mnath30) [inline src/components/carousel/index.css:19]: If you do not need these css then it is better to remove those instead of commenting them

**Review by mnath30 — APPROVED** [review]
Code is clean 
Everything is broken down into small components which makes it easier to read and review

File: src/components/carousel/index.css (hunk 1 above)
Comment (Vishruta-Patil) [reply src/components/carousel/index.css:19]: Thanks for the review, I have commented those lines of code as I need them for future use.

**Pull request merged**